### PR TITLE
OCPBUGS-8006: More frequent readiness check for new cluster only

### PR DESCRIPTION
**Pull request by jc-rh**
Also removed repetitive logs.

### CODE REVIEW

File: controllers/managedclusterForCGU_controller.go [hunk 1]
```diff
@@ -134,13 +131,19 @@ func (r *ManagedClusterForCguReconciler) Reconcile(ctx context.Context, req ctrl
 		}
 	} else {
 		// availableCondition is false or unknown, cluster is not ready
-		r.Log.Info("cluster is not ready", "RequeueAfter:", clusterStatusCheckRetryDelay)
```
Comment (nishant-parekh) [inline controllers/managedclusterForCGU_controller.go:137]: do you want to add this log to the new common function, probably helps with debugging?

**Review by nishant-parekh — APPROVED** [review]
just a small suggestion. but looks good otherwise

File: controllers/managedclusterForCGU_controller.go (hunk 1 above)
Comment (jc-rh) [reply controllers/managedclusterForCGU_controller.go:137]: Potentially. Normally this log is just flooding our log file, especially with this change. There seem to be only 2 levels with our logger: info and error.

**Pull request merged**